Source/ROLib/ModelDef/ROLModelDefinition.cs

```diff
@@ -1,1096 +1,1115 @@
-﻿using System;

```

Comment:
Why is this *entire file* diff'd?  Not sure how to review this.

Reply:
I imagine inconsistent line endings were normalized